### PR TITLE
Bug fixes and new operations

### PR DESCRIPTION
**Pull request by grctest**
Localized 3 new operations
Fixed the create/update asset beautify function
Fixed issue #273

### CODE REVIEW

**Comment by grctest** [comment]
Needs an update to bitsharesjs https://github.com/bitshares/bitsharesjs/issues/123

**Comment by sschiessl-bcp** [comment]
> Needs an update to bitsharesjs [bitshares/bitsharesjs#123](https://github.com/bitshares/bitsharesjs/issues/123)

Please test again

File: package.json [hunk 1]
```diff
@@ -1,105 +1,105 @@
 {
-  "name": "beet",
```
Comment (sschiessl-bcp) [inline package.json:2]: intentional change in this file?

Comment (sschiessl-bcp) [reply package.json:2]: @grctest

Comment (grctest) [reply package.json:2]: Sorry, I think I save & committed this file changing just rc1 to rc2, but I had a different spacing enabled switching between projects I'm working on. I'll revert the change to improve readability.

**Comment by grctest** [comment]
I have successfully broadcast an OSO enabled operation on the blockchain: https://blocksights.info/#/operations/1.11.1291557405

```
{
  "fee": {
    "amount": 48260,
    "asset_id": "1.3.0"
  },
  "seller": "1.2.1811495",
  "amount_to_sell": {
    "amount": 90909,
    "asset_id": "1.3.0"
  },
  "min_to_receive": {
    "amount": 100,
    "asset_id": "1.3.5649"
  },
  "expiration": "2023-11-25T01:20:47",
  "fill_or_kill": false,
  "extensions": {
    "on_fill": "[[0,{\"fee_asset_id\":\"1.3.0\",\"spread_percent\":100,\"size_percent\":10000,\"expiration_seconds\":1000000000,\"repeat\":true,\"extensions\":[]}]]",
    "on_fill_object": [
      {
        "which": 0,
        "data_object": {
          "fee_asset_id": "1.3.0",
          "spread_percent": 100,
          "size_percent": 10000,
          "expiration_seconds": 1000000000,
          "repeat": true,
          "extensions": []
        }
      }
    ]
  }
}
```

```
[
  1,
  "1.7.531970513"
]
```

**Comment by grctest** [comment]
Failed to broadcast a limit order update: https://github.com/bitshares/bitsharesjs/issues/123#issuecomment-1843041870

**Comment by grctest** [comment]
Successfully broadcast a limit order update operation through this BEET branch: https://blocksights.info/#/operations/1.11.1294152022

**Comment by grctest** [comment]
@sschiessl-bcp Ready to merge into the develop branch

**Review by sschiessl-bcp — APPROVED** [review]

**Comment by sschiessl-bcp** [comment]
I accidently squashed now ... I assume its not a dealbreaker and will leave it

**Pull request merged**